platform/kobo/nickel.sh

```diff
@@ -17,13 +18,19 @@ unset LC_ALL STARDICT_DATA_DIR EXT_FONT_DIR
 unset KO_DONT_GRAB_INPUT
 unset FBINK_FORCE_ROTA
 
-# Ensures fmon will restart. Note that we don't have to worry about reaping this, nickel kills on-animator.sh on start.
-(
-    if [ "${PLATFORM}" = "freescale" ] || [ "${PLATFORM}" = "mx50-ntx" ] || [ "${PLATFORM}" = "mx6sl-ntx" ]; then
-        usleep 400000
-    fi
-    /etc/init.d/on-animator.sh
-) &
+if [ -e "/etc/init.d/display-init.sh" ]; then
+    /etc/init.d/display-init.sh
```

Comment:
I'd rather just run `animator.sh /etc/images/${PRODUCT}-on-*.raw.gz &` here, not suuuuper confident about re-running pickel init at runtime, among other things ;).